src/containers/HeaderContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { HeaderView } from '../views/HeaderView';
+
+function mapStateToProps(state) {
+  return {
+    isTagMode: state.isTagMode,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return null;
+}
```

Comment:
we don't always have to define a `mapDispatchToProps`. if unused, we can pass in `null` like you did below, or omit it fully:

```js
*Container = connect(mapStateToProps)(HeaderView);
```

the same applies to the opposite scenario with `mapStateToProps`, if we just wanted to pass in a `mapDispatchToProps`. though we have to explicitly pass `null` as the first argument in these cases:

```js
*Container = connect(null, mapDispatchToProps)(HeaderView);
```